aio/content/marketing/contributors.json

```
@@ -840,5 +840,13 @@
     "bio": "Emma is a Developer Advocate at Google. She is passionate about good user experiences and design.",
     "groups": ["Angular"],
     "lead": "mgechev"
+  },
+  "kreuzercode": {
+    "name": "Kevin Kreuzer",
+    "picture": "kevin-kreuzer.jpg",
+    "twitter": "kreuzercode",
+    "website": "kreuzercode.com",
+    "bio": "Kevin is a passionate freelance front-end engineer and Google Developer Expert based in Switzerland. He is a JavaScript enthusiast and fascinated by Angular. Kevin always tries to learn new things, expand his knowledge, and share it with others in the form of blog posts, workshops, podcasts, or presentations. He is a writer for various publications and the most active writer on Angular in-depth in 2019. Contributing to multiple projects and maintaining 7 npm packages, Kevin is also a big believer in open source. Furthermore, Kevin is a big football fan. Since his childhood, he supports Real Madrid, which you will notice in a lot of his blog posts and tutorials.",
```

Hi Kevin! I just rebased for you but then noticed a little edit I'd suggest. I already have your branch locally, so would you mind if I just made this change?
```suggestion
    "bio": "Kevin is a passionate freelance front-end engineer and Google Developer Expert based in Switzerland. He is a JavaScript enthusiast and fascinated by Angular. Kevin always tries to learn new things, expand his knowledge, and share it with others in the form of blog posts, workshops, podcasts, or presentations. He is a writer for various publications and the most active writer on Angular in-depth in 2019. Contributing to multiple projects and maintaining 7 npm packages, Kevin is also a big believer in open source. Furthermore, Kevin is a big football fan. Since his childhood, he has supported Real Madrid, which you might notice in a lot of his blog posts and tutorials.",
```

---

Epic! Yes, of course. Thanks a lot. 👍

---

Great! I just updated it. So now we just have to get the tests all passing and marketing approval, then we can move forward. I'll help it through those steps. Feel free to tag me if you have any questions. Thanks for everything!😊